aptos-move/framework/aptos-framework/sources/primary_wallet.move

```diff
@@ -0,0 +1,110 @@
+/// This defines the module for interacting with primary wallets of accounts/objects, which have deterministic addresses
+module aptos_framework::primary_wallet {
+    use aptos_framework::create_signer;
+    use aptos_framework::fungible_asset::{Self, FungibleAsset, FungibleAssetMetadata, FungibleAssetWallet};
+    use aptos_framework::object::{Self, Object, ConstructorRef, DeriveRef};
+
+    use std::signer;
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    /// Resource stored on the fungible asset metadata object to allow creating primary wallets for it.
+    struct PrimaryWalletSupport has key {
```

Comment:
maybe rename as `DeriveRefPod`. Support as a resource looks weird.

Reply:
Peas in a pod? Let's rename things all in one go in a followup PR